nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -38,12 +38,23 @@ def __init__(self):
         self.phases = queue.Queue()
 
     def go_to_patient(self):
-        # Extract room number and bed number from goal (bell)
-        status = self.navigate_task.execute(self.room, "bed")
+        rospy.loginfo("Nightingale Mission Planner going to patient")
+        # Assume door is open
+        # TODO: first go to doorside then bedside when door opening added
+        status = self.navigate_task.execute(self.room, "bedside")
```

Comment:
If the name is changed from `bed` to `bedside` here, the room_pose_lookup.yaml also needs to be changed

Reply:
ill address this once i add the lookup funcationality to room runner